Artifact missing in offline build

The `applications` build fails when run offline (`mvn -o -P local-dev [...]`).
The generated error is raised while building `applications/apputil/apputil-plugins/org.csstudio.utility.treemodel.test`:  
```
[ERROR] The following artifacts could not be downloaded:
[ERROR]   osgi.bundle,org.custommonkey.xmlunit,1.3.0.v201605172130
```
The xmlunit (v.1.5.0) bundle is included in maven-osgi-bundles but is not picked up by the treemodel plugin.

--- a/applications/apputil/apputil-plugins/org.csstudio.utility.treemodel.test/META-INF/MANIFEST.MF
+++ b/applications/apputil/apputil-plugins/org.csstudio.utility.treemodel.test/META-INF/MANIFEST.MF
@@ -7,5 +7,5 @@ Fragment-Host: org.csstudio.utility.treemodel;bundle-version="1.3.0"
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Vendor: Desy <css-developer@desy.de>, Bastian Knerr
 Import-Package: com.google.common.base,
- com.google.common.collect,
- org.custommonkey.xmlunit
+ com.google.common.collect
+Require-Bundle: xmlunit

